Makefile

```diff
@@ -3,6 +3,11 @@
 NOJESSIE ?= 1
 NOSTRETCH ?= 0
 
+PLATFORM_PATH := platform/$(if $(PLATFORM),$(PLATFORM),$(CONFIGURED_PLATFORM))
```

Comment:
When building the images, the PLATFOMR_PATH will be equal to platform/. We add the value in .platform, but the variable PLATFORM and the variable CONFIGURED_PLATFORM are empty here. It is no harm, if someone uses it, may have an issue.
For example:
make PLATFORM=vs configure
**make target/sonic-vs.img**